sdk/core/core-amqp/src/errors.ts

```diff
@@ -560,6 +553,19 @@ export function translate(err: AmqpError | Error): MessagingError {
 
   let error: MessagingError = err as MessagingError;
 
+  // OperationTimeoutError is retryable as it indicates potential problem with unexpected delays
+  // and not an issue with the Messaging process
+  if (
+    err instanceof OperationTimeoutError ||
+    // instanceof checks on custom Errors doesn't work without manually setting the prototype within the error.
+    // Must do a name check until OperationTimeoutError is updated, and that doesn't break compatibility
+    // https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
+    (err as Error).name === "OperationTimeoutError"
+  ) {
+    error.retryable = true;
```

Comment:
Let's also add `OperationTimeoutError` to the list of retryable errors exported from `core-amqp`. This will help any consumer of `core-amqp` who is inspecting errors without using `translate()`